apollo-router/src/plugins/telemetry/config_new/attributes.rs

```diff
@@ -264,6 +272,78 @@ impl DefaultForLevel for SubgraphAttributes {
     }
 }
 
+#[derive(Deserialize, JsonSchema, Clone, Default, Debug)]
+#[serde(deny_unknown_fields, default)]
+pub(crate) struct ConnectorHttpAttributes {
```

Comment:
I think we should put this in subgraph attributes. Let's only keep subgraph attributes for now, and as it's namespaced by connector. It will be enough, I also think we shouldn't have both `connector.http.method` and `subgraph.http.method` we should just keep the subgraph one for now. If people wants to know if it's a connector or a subgraph I would suggest adding in `SubgraphSelelector` something like `service_kind` which could be `connector` or `subgraph`. (I don't think `service_kind` is the right naming but you got the idea).

Reply:
The issue here is that the relationship between subgraph and connector is not 1-to-1. There can be a number of connectors on a given subgraph, and there can be multiple calls to each connector per GraphQL request. So a connector isn't really a "kind" of subgraph - the subgraph service lifecycle stage is not invoked for connectors. The lifecycle is really now like this:

```
                 ------ subgraph service ------ http client service
                |
fetch service --|
                |                                ------- http client service
                 ------ connector service ------|
                                                 ------- http client service
```

The fetch service and connector service aren't currently exposed as plugin lifecycle hooks, but likely will be.

Reply:
Hmm indeed, I didn't know we could have several http calls for a connector. Any idea @BrynCooke ?

Reply:
I think what's here is the way forward, and then we add `http_client` service support to the telemetry plugin.

I know it's a big job, but it feels like the right thing to do.